modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -51,17 +63,19 @@ class AlephClient {
    *   The base url for the Aleph end-point.
    * @param $base_url_rest
    *    The base url for the Aleph REST end-point.
-   *
    * @param $main_library
    *    The main library. For example ICE01.
-   *
    * @param $filter_institution
    *    The institution filter to set. ICE53 for example.
+   * @param $branches
+   *    Filter to certain branches.
```

Comment:
I would prefer that the argument was kept in the strictest format possible as long a possible. In that regard `$branches` should be `string[]` all the way until it is used as a request parameter in `AlephClient->getItems()`

If we do not change this then at least update the argument for the constructor to detail level of the `$branches` property.